usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -313,6 +308,20 @@ def handle(self, *args, **options):
             "spark.sql.jsonGenerator.ignoreNullFields": "false",  # keep nulls in our json
         }
 
+        # Resolve Parameters
+        destination_table = options["destination_table"]
+        table_spec = TABLE_SPEC[destination_table]
+        self.destination_database = options["alt_db"] or table_spec["destination_database"]
+        self.destination_table_name = options["alt_name"] or destination_table.split(".")[-1]
+        load_query = table_spec["source_query"]
+
+        if options["incremental"]:
+            if not table_spec["source_query_incremental"]:
```

Comment:
Since `source_query_incremental` is not a boolean, this statement is harder to read than it should be. Can you change it to `if table_spec["source_query_incremental"] is not None`?

Reply:
I took a closer look at this. Both options would actual result in a `KeyError`. Updated the line to:
```
if table_spec.get("source_query_incremental") is not None:
```